build.sbt

```diff
@@ -1,5 +1,15 @@
 import Dependencies.Library
 
+val scala212 = "2.12.13"
+val scala213 = "2.13.4"
+
+inThisBuild(
+  List(
+    scalaVersion := scala212,
+    version := "0.1.0"
```

Comment:
Isn't the version now handled automatically? Or it's not set up in this repo, as with many other things that missing from this repo 🤣

Reply:
SBT needs a version to publish, this is not only for CI but also for local development and integration with other projects. The version cannot be the same as the old/previous, especially since it can be considered a breaking change given the updated dependencies.